### ISSUE TITLE
Add support for arbitrary electric namespaces

### ISSUE THREAD

**Issue by milelo** [milelo/electric-hiccup#13]
eg
* hyperfiddle.electric-svg 
* hyperfiddle.electric-ui4

**Comment by milelo** [comment]
You can now use namespaced keywords as hiccup tag-forms. so for example:

```clojure
 [hyperfiddle.electric-svg :as svg]
```

and use it:

```clojure
#electric-hiccup
[:svg/svg {:viewBox "0 0 300 100"}
       [:svg/circle {:cx 50 :cy 50 :r (+ 30 offset)
                     :style {:fill "#af7ac5 "}}]
       [:svg/g {:transform
                (str "translate(105,20) rotate(" (* 3 offset) ")")}
        [:svg/polygon {:points "30,0 0,60 60,60"
                       :style {:fill "#5499c7"}}]]
       [:svg/rect {:x 200 :y 20 :width (+ 60 offset) :height (+ 60 offset)
                   :style {:fill "#45b39d"}}]]
```

**Comment by milelo** [comment]
Available in: df87af211f0f16f1f616596a478d9736d20f25d9

**Comment by milelo** [comment]
Released in:
```clojure
        milelo/electric-hiccup {:git/url "https://github.com/milelo/electric-hiccup"
                                :git/tag "v0.4.0-alpha"
                                :git/sha "f29103a"}
```
